content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -114,13 +114,13 @@ the name of the MySQL Service defined above, and WordPress will access the datab
       
 3. Add them to `kustomization.yaml` file.
 
-      ```shell
-      cat <<EOF >>./kustomization.yaml
-      resources:
-        - mysql-deployment.yaml
-        - wordpress-deployment.yaml
-      EOF
-      ```
+```shell
+cat <<EOF >>./kustomization.yaml
+resources:
+  - mysql-deployment.yaml
+  - wordpress-deployment.yaml
+EOF
```

Comment:
@aburomaissae , You could probably remove the `EOF`. With these changes, are the instructions clear to you and does the example work for you?